install/debian-installation-guide.md

```diff
@@ -0,0 +1,47 @@
+# Install Kata Containers on Debian
+
+These install instructions are based on the Ubuntu packages, but we need to install an updated version of `librbd1`.
+Tested only on Debian 9 - Stretch (x86_64).
+
+1. Dependencies
+
+  We need a newer version of librdbd1 to satisfy requirements: https://packages.debian.org/sid/librbd1
+
+  Add `unstable` repo to `/etc/apt/sources.list.d/unstable.list` sources list:
+  
+  ```bash
+  sudo sh -c "echo '# for unstable packages
+  deb http://ftp.debian.org unstable main contrib non-free
+  deb-src http://ftp.debian.org unstable main contrib non-free' > /etc/apt/sources.list.d/unstable.list"
+  ```
+  
+  Set it with a lower priority than stable, edit/create `/etc/apt/preferences.d/unstable`:
+  
+  ```bash
+  $ sudo sh -c "echo 'Package: *
+  Pin: release a=unstable
+  Pin-Priority: 10' >> /etc/apt/preferences.d/unstable"
+  ```
+
+  This ensures that apt doesn't try to pull unstable updates for all your packages!
+
+  Update and install package
+
+  ```bash 
+  $ sudo apt update && sudo apt install -y -t unstable librbd1
+  ```
+
+2. Install the Kata Containers components with the following commands:
+
+   ```bash
+   $ ARCH=$(arch)
+   $ sudo sh -c "echo 'deb http://download.opensuse.org/repositories/home:/katacontainers:/releases:/${ARCH}:/master/Debian_9.0/$(lsb_release -rs)/ /' > /etc/apt/sources.list.d/kata-containers.list"
+   $ curl -sL  http://download.opensuse.org/repositories/home:/katacontainers:/releases:/${ARCH}:/master/Debian_9.0/$(lsb_release -rs)/Release.key | sudo apt-key add -
+   $ sudo -E apt-get update
```

Comment:
Could you remove the hard-coded `Debian_9.0` by doing:

```sh
$ source /etc/os-release
    :
$ sudo sh -c "echo 'deb http://download.opensuse.org/repositories/home:/katacontainers:/releases:/${ARCH}:/master/$(lsb_release -si)_${VERSION_ID}/$(lsb_release -rs)/ /' > /etc/apt/sources.list.d/kata-containers.list"
  :
```

That's getting a little crazy as there are lots of variable. But of course, the user only needs to copy and paste the commands (and it means we won't need to update this when new versions of Debian are released (I hope ;)

Reply:
Good point. Let's leave this "as-is" for now then.

/cc @marcov for thoughts on the repo though.

Reply:
Our OBS projects use the primary release number atm so for Ubuntu Xenial we have:

- https://download.opensuse.org/repositories/home:/katacontainers:/releases:/x86_64:/master/xUbuntu_16.04/

But the latest version of that release is [16.04.5](http://releases.ubuntu.com/16.04/). Based on this, I think `Debian_9` makes sense.

Reply:
Updated.